internal/v1/api.yaml

```diff
@@ -1942,20 +1942,36 @@ components:
           description: |
             Optional flag to use rhc to register the system, which also always enables Insights.
     OpenSCAP:
+      oneOf:
+        - $ref: '#/components/schemas/OpenSCAPProfile'
+        - $ref: '#/components/schemas/OpenSCAPCompliance'
+    OpenSCAPCompliance:
+      type: object
+      required:
+        - policy_id
+      properties:
+        policy_id:
+          type: string
+          format: uuid
+          example: 'fef25b3c-b970-46da-a4e1-cc4d855b98dc'
+          description: |
+            The compliance policy ID, either policy_id or profile_id must be set.
+    OpenSCAPProfile:
       type: object
       required:
         - profile_id
       properties:
         profile_id:
           type: string
           example: "xccdf_org.ssgproject.content_profile_cis"
-          description: "The policy reference ID"
+          description: |
+            The profile reference ID, either the profile_id or policy_id must be set.
         profile_name:
           type: string
-          description: "The policy type"
+          description: "The profile type"
         profile_description:
           type: string
-          description: "The longform policy description"
+          description: "The longform profile description"
     CustomRepository:
```

Comment:
I find documentation string confusing. I think there should be an overall text describing the difference between the two schemas, and then description for each one of them of what exactly the ID is and where you get it.